pkg/manifest/manifest.go

```diff
@@ -98,6 +110,69 @@ func (m *Manifest) UnmarshalJSON(in []byte) error {
 	return validateResourceId(m.id)
 }
 
+func (m *Manifest) Include(excludeIdentifier string, includeTechPreview bool, profile string, capabilities ClusterCapabilities) error {
```

Comment:
If we want to use a single "should we include this?" method both for load-time filtering (cluster profile, etc.) and apply-time filtering (capabilities, etc.), we should make these arguments pointers, and internally only filter with the non-`nil` terms.

Reply:
Fixed in https://github.com/openshift/library-go/commit/e35ac895df110ebbd4157d2e853f4715dd501ad6